Page Object/Elements/webTablesPage.ts

```diff
@@ -1,6 +1,23 @@
 import { basePage } from "../basePage";
 export class WebTablesPage extends basePage {
-    constructor (page) {
-        super (page)
-    }
-}
\ No newline at end of file
+  constructor(page) {
+    super(page);
+  }
+  Buttons = {
+    webTalesButton: this.page.locator("li").filter({ hasText: "Web Tables" }),
```

Comment:
Is it possible to use page.getByText() in this situation?

Reply:
> Is it possible to use page.getByText() in this situation?

Yes, it will be more concise